src/components/pages/manager/ManagerItem.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable react/self-closing-comp */
+import React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  Container, Button, Row, Col,
+} from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import DestinationItem from '../requests/DestinationItem';
+
+const ManagerItem = ({ item, user, email }) => {
```

Comment:
You do not need this file. Reuse the `searchBar` component in `components/pages/requests` instead of repeating the same code. You can give it a condition for rendering the Username field since it's the only thing changing here.